contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/EditBooleanMapModal.vue

```diff
@@ -91,16 +94,27 @@
          */
         selectedValues: {},
         changed: false,
+        hasMixedCategories: false,
       };
     },
     computed: {
-      ...mapGetters('contentNode', ['getContentNodes']),
+      ...mapGetters('contentNode', ['getContentNodes', 'getContentNode']),
       nodes() {
         return this.getContentNodes(this.nodeIds);
       },
       isTopicSelected() {
         return this.nodes.some(node => node.kind === ContentKindsNames.TOPIC);
       },
+      canSave() {
+        if (this.hasMixedCategories) {
+          return Object.values(this.selectedValues).some(value => value.length > 0);
```

Comment:
I just thought about it, and the condition should be 
```suggestion
          return Object.values(this.selectedValues).some(value => value.length === this.nodes.length);
```

As this should be the condition to say that there is at least one selected value.

Apologies for the misdirection previously